grype/search/cpe_test.go

```diff
@@ -215,6 +215,71 @@ func TestFindMatchesByPackageCPE(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "fallback to package version",
```

Comment:
that's right -- this is something that grype has been doing (this PR does not change this behavior). However, I added a test case for extra measure since I was changing code around this (I didn't see an explicit one already).